tensorflow-cnn/cnn_example_gpu.py

```diff
@@ -0,0 +1,162 @@
+# Experiment setup
+import argparse
+import datetime
+import numpy
+import time
+import scipy
+import sigopt
+import scipy.io
+import tensorflow as tf
+import math
+from skimage.color import rgb2gray
+from sklearn import preprocessing
+from sklearn.cross_validation import train_test_split
+
+conn = sigopt.Connection(client_token=client_token)
```

Comment:
Ah we need the snippet to create the SigOpt experiment object here.  Also I think in the cpu example we just are assuming the connection will grab the API creds from the env vars, so let's make that consistent here as well

Otherwise looks good to me!